examples/js/webpack.config.js

```diff
@@ -18,11 +18,5 @@ module.exports = {
       title: 'Spectacle JavaScript Demo',
       template: `./index.html`
     })
-  ],
-  resolve: {
-    ...base.resolve,
-    alias: {
-      spectacle: path.resolve(__dirname, '../../src')
-    }
-  }
```

Comment:
No longer need to resolve here, we'll just use pnpm workspace settings (and we'll use the actual build files instead of src, which gives a more accurate depiction of real-world use)